build.gradle

```diff
@@ -144,3 +151,23 @@ uploadArchives {
 }
 
 apply from: 'changelog.gradle'
+
+dependencies {
+    testCompile mcTesterDep
+    testCompile "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    testCompile "org.jetbrains.kotlinx:kotlinx-coroutines-core:0.23.4"
+    testCompile "org.jetbrains.kotlin:kotlin-reflect:1.2.51"
+}
+
+kotlin {
+    experimental {
+        coroutines 'enable'
+    }
+}
+
```

Comment:
Add a line `license.include **/*.kt` to auto format kotlin files for their license.

Reply:
Fixed